docs/examples/shading/plot_shaded_fraction1d_ns_hsat_example.py

```diff
@@ -0,0 +1,185 @@
+"""
+shaded_fraction1d N-S horizontal single-axis example
+====================================================
+
+This example illustrates how to calculate the shaded fraction of three rows
+in an N-S HSAT configuration.
+"""
+
+# %%
+# :py:func:`pvlib.shading.shaded_fraction1d` exposes a useful method for the
+# calculation of the shaded fraction of the width of a solar collector. Here,
+# the width is defined as the dimension perpendicular to the axis of rotation.
+# This method for calculating the shaded fraction also applies to fixed-tilt
+# systems with little changes.
+#
+# Reading its documentation is recommended to understand the parameters and
+# the method capabilities.
+#
+# Let's start by obtaining the true-tracking angles for each of the rows and
+# limiting the angles to the range of -50 to 50 degrees. This decision is
+# done to allow significant shade to be used as an example.
+#
+# Key functions used in this example are:
+#
+# 1. :py:func:`pvlib.tracking.singleaxis` to calculate the tracking angles.
+# 2. :py:func:`pvlib.shading.projected_solar_zenith_angle` to calculate the
+#    projected solar zenith angle.
+# 3. :py:func:`pvlib.shading.shaded_fraction1d` to calculate the shaded
+#    fractions.
+#
+# .. sectionauthor:: Echedey Luis <echelual (at) gmail.com>
+
+import pvlib
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from matplotlib.dates import DateFormatter
+
+# Define the solar system parameters
+latitude, longitude = 28.51, -13.89
+altitude = pvlib.location.lookup_altitude(latitude, longitude)
+
+axis_tilt = 3  # degrees
```

Comment:
Could you add a comment about what this represents?

Reply:
Let me know your opinion on `axis_tilt = 3  # degrees, positive is upwards in the axis_azimuth direction`